bin/rucio

```diff
@@ -933,6 +933,8 @@ def download(args):
     item_defaults['transfer_timeout'] = args.transfer_timeout
     item_defaults['no_resolve_archives'] = args.no_resolve_archives
     archive_did = args.archive_did
+    if archive_did:
+        logger.warning("Archives are treated transparently. --archive-did option is beign obsoleted.")
```

Comment:
typo `beign`

Reply:
@TomasJavurek Please fix type and also add a `# TODO` that we should remove the `--archive-did` completely in the future.